### PR TITLE
Change text to full_text

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

**Comment by rodrigopavezi** [comment]
@Rise-Vision/content tested on my local player and it worked. cheers

**Comment by rodrigopavezi** [comment]
[Show all characters of a tweet [3]](https://trello.com/c/yS7VvNil/5098-show-all-characters-of-a-tweet-3)

**Comment by rudibravo** [comment]
@rodrigopavezi should this PR wait the twitter module roll out?

**Comment by rodrigopavezi** [comment]
@rudibravo I thought about the problem with release dependency on the module so I think it is best to make the component back compatible.

@sheadarlison looking into the font size now. cheers

**Comment by sheadarlison** [comment]
@rodrigopavezi just on the font size, if there are 140 or less characters it should remain the size it currently is. If there are more than 140 characters it should decrease to fit the container.

**Comment by rodrigopavezi** [comment]
@rudibravo can you review it again? cheers

**Comment by sheadarlison** [comment]
@rodrigopavezi it looks like on portrait parts of tweets are still getting cut off. See attached images. 

5e732155-6e8b-43d5-80d3-2b7f6546059b
d41eef49-68da-4023-8f10-b99c58a6f12a
810f570c-4bd4-4247-a12e-8790b8a74053

**Comment by sheadarlison** [comment]
@rodrigopavezi It looks good but did you change the URL color? It used to be blue. See images below 

OLD
presentation_template_rva_twitterfullscreentemplatev3

NEW
26ae1116-1509-4206-ae5f-40b804af90ac

File: src/tweet.js [hunk 1]
```diff
@@ -237,21 +237,28 @@ export default class Tweet {
...
 
     for (const urlIndex in tweetData.entities.urls) {
       const url = tweetData.entities.urls[urlIndex].url;
-      textWithoutLinks = textWithoutLinks.replace(url, '');
+      textWithLinks = textWithLinks.replace(url, this._getLinkSpan(url).outerHTML);
+    }
+    if (tweetData.extended_entities) {
+      for (const mediaIndex in tweetData.extended_entities.media) {
+        const url = tweetData.extended_entities.media[mediaIndex].url;
+        textWithLinks = textWithLinks.replace(url, this._getLinkSpan(url).outerHTML);
```
Comment (rudibravo) [inline src/tweet.js]: @rodrigopavezi is there a use case to display the media as a link? I don't think it aggregates much to the people seeing the twitter on displays.
Besides as per the example on the [twitter documentation](https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/extended-entities-object), all media will point to the same url, which is the tweet url.

Comment (sheadarlison) [reply src/tweet.js]: @rudibravo Are you suggesting just hiding the URL?

Comment (rudibravo) [reply src/tweet.js]: @sheadarlison Yes, I don't see much value on showing some pictures URLs.

Comment (sheadarlison) [reply src/tweet.js]: @rudibravo yeah I agree with that. @rodrigopavezi could you hide image urls but make sure links to other websites still show with the blue color.

Comment (rodrigopavezi) [reply src/tweet.js]: @sheadarlison @rudibravo sure, that makes sense. cheers

**Comment by rodrigopavezi** [comment]
@sheadarlison removed the media links. Please review. cheers

**Comment by sheadarlison** [comment]
@rodrigopavezi looks good. Thanks

**Pull request merged**